### PR TITLE
Update hybrid-configuration-wizard.md

### PR DESCRIPTION
**Pull request by mukeshHCW**
The role required to run HCW must be global admin in M365. This is the correction made here. 

_Please reach out to mukkum@microsoft.com for any clarification_

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Exchange/ExchangeHybrid/hybrid-configuration-wizard.md [hunk 1]
```diff
@@ -51,7 +51,7 @@ The general phases of the hybrid deployment configuration process are:
...
+2. **Testing account credentials**: Designated on-premises and Microsoft 365 or Office 365 hybrid management accounts access the on-premises and Exchange Online organizations to gather prerequisite verification information and to make organization parameter configuration changes to enable hybrid deployment functionality. The Hybrid Configuration wizard checks that the accounts have the appropriate credentials and can connect to the on-premises and Exchange Online organizations. The hybrid deployment management accounts for the on-premises must be member of the Organization Management role group and must be Global Admin for Microsoft 365 or Office 365 organizations for the Hybrid Configuration wizard to complete these tasks successfully.
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeHybrid/hybrid-configuration-wizard.md:54]:
```suggestion
2. **Testing account credentials**: Designated on-premises and Microsoft 365 or Office 365 hybrid management accounts access the on-premises and Exchange Online organizations to gather prerequisite verification information and to make organization parameter configuration changes to enable hybrid deployment functionality. The Hybrid Configuration wizard checks that the accounts have the appropriate credentials and can connect to the on-premises and Exchange Online organizations. The hybrid deployment management accounts for on-premises organizations must be members of the Organization Management role group and must be Global Admins for Microsoft 365 or Office 365 organizations for the Hybrid Configuration wizard to complete these tasks successfully.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@Kateyanne PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**